### ISSUE TITLE
Renovate fails with the pip_requirements manager

### ISSUE THREAD

**Issue by lorisepita** [renovatebot/renovate#10499]
**How are you running Renovate?**

- [ ] WhiteSource Renovate hosted app on github.com
- [X] Self hosted

If using the hosted app, please skip to the next section.
Otherwise, if self-hosted, please complete the following:

Please select which platform you are using:

- [ ] Azure DevOps (dev.azure.com)
- [ ] Azure DevOps Server
- [ ] Bitbucket Cloud (bitbucket.org)
- [ ] Bitbucket Server
- [ ] Gitea
- [ ] github.com
- [ ] GitHub Enterprise Server
- [ ] gitlab.com
- [X] GitLab self-hosted

Renovate version: 25.41.7

**Describe the bug**

Renovate cannot update the `requirements.txt` file because of an exception.

**Relevant debug logs**

<details><summary>Click me to see logs</summary>

```
DEBUG: Failed to parse newContent (repository=project, packageFile=requirements.txt, branch=renovate/python-pip)
       "manager": "pip_requirements",
       "err": {
         "message": "extractPackageFile is not a function",
         "stack": "TypeError: extractPackageFile is not a function\n    at confirmIfDepUpdated (/usr/src/app/node_modules/renovate/lib/workers/branch/auto-replace.ts:27:30)\n    at Object.doAutoReplace (/usr/src/app/node_modules/renovate/lib/workers/branch/auto-replace.ts:190:19)\n    at Object.getUpdatedPackageFiles (/usr/src/app/node_modules/renovate/lib/workers/branch/get-updated.ts:99:19)\n    at Object.processBranch (/usr/src/app/node_modules/renovate/lib/workers/branch/index.ts:313:17)\n    at Object.writeUpdates (/usr/src/app/node_modules/renovate/lib/workers/repository/process/write.ts:38:17)\n    at Object.update (/usr/src/app/node_modules/renovate/lib/workers/repository/process/extract-update.ts:129:11)\n    at Object.renovateRepository (/usr/src/app/node_modules/renovate/lib/workers/repository/index.ts:50:17)\n    at Object.start (/usr/src/app/node_modules/renovate/lib/workers/global/index.ts:111:7)\n    at /usr/src/app/node_modules/renovate/lib/renovate.ts:16:22"
       }
DEBUG: No newUpgrade (repository=project, packageFile=requirements.txt, branch=renovate/python-pip)
       "manager": "pip_requirements"
 WARN: Error updating branch: update failure (repository=project, branch=renovate/python-pip)
```

</details>

<details><summary>Click me to see relevant configuration</summary>

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    "group:all"
  ],  
  "updateLockFiles": true,
  "gitLabAutomerge": false,
  "dependencyDashboard": true,
  "baseBranches": ["dev"],
  "packageRules": [
    {   
      "matchManagers": ["poetry"],
      "groupName": "python poetry dependencies",
      "groupSlug": "python-poetry"
    },  
    {   
      "matchManagers": ["pip_requirements"],
      "groupName": "python pip dependencies",
      "groupSlug": "python-pip"
    },  
    {   
      "matchLanguages": ["docker"],
      "groupName": "docker dependencies",
      "groupSlug": "docker"
    }   
  ]
}
```

</details>

<details><summary>Click me to see requirements.txt</summary>

```
flask-admin==1.5.7
flask-cors==3.0.6
flask-graphql==1.4.1
flask-login==0.4.1
flask-mail==0.9.1
flask-sqlalchemy==2.3.2
flask-wtf==0.14.2
flask==1.0.2
```

</details>

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [X] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Comment by viceice** [comment]
fixed in [v24.47.0](https://github.com/renovatebot/renovate/releases/tag/25.47.0)

**Comment by viceice** [comment]
Duplicate of #10485
